### PR TITLE
[Nginx Otel] Fix body structured field in dashboards

### PR DESCRIPTION
**Pull request by rogercoll**
## Proposed commit message

Use `body.structured` namespace instead of `attributes` for dashboards.

In addition, some fields for the `nginx_ingress_controller.error` event has been mapped to the proposed semantic convetnion version: https://github.com/open-telemetry/semantic-conventions/pull/982 (without the `error` prefix)

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [ ] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices) 

## Author's Checklist

- [ ] 

## Related issues

-

### CODE REVIEW

File: packages/nginx_ingress_controller_otel/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.0.2"
+  changes:
+    - description: Use body.structured field for dashboards
+      type: enhancement
```
[the remark this answers is not in the document]
Comment (rogercoll) [reply packages/nginx_ingress_controller_otel/changelog.yml]: Definitely a bugfix, changed https://github.com/elastic/integrations/pull/11625/commits/623e5955e0face394049b28a1a7d52a0f32671cb

File: packages/nginx_ingress_controller_otel/docs/README.md [hunk 2]
```diff
@@ -8,16 +8,16 @@ instances. It can parse access and error logs created by the ingress.
 The integration was tested with the Nginx Ingress Controller v0.30.0 and v0.40.2. The log format is described
 [here](https://github.com/kubernetes/ingress-nginx/blob/nginx-0.30.0/docs/user-guide/nginx-configuration/log-format.md).
 
-**EDOT collector supported versions:** 8.16.0-SNAPSHOT
+**EDOT collector supported versions:** 8.16.0
```
Comment (ishleenk17) [inline packages/nginx_ingress_controller_otel/docs/README.md:11]: We are mentioning collector version as 8.16 and kibana.version in manifest as 9.0.0
Do we plan to support this from 8.16.0 onwards or 9.0.0 onwards ?

Comment (rogercoll) [reply packages/nginx_ingress_controller_otel/docs/README.md:11]: Thanks @ishleenk17, just changed the Kibana version to 8.16.0 so its aligned with the EDOT collector one: https://github.com/elastic/integrations/pull/11625/commits/faa1df794d25a3d679cc86cd7a6477c834d0e702

>  Do we plan to support this from 8.16.0 onwards or 9.0.0 onwards ?

8.16.0 but the integration is on tech preview, there might be some breaking changes depending on the upstream Semantic convention decisions.

Comment (ishleenk17) [reply packages/nginx_ingress_controller_otel/docs/README.md:11]: Breaking changes might always be part of these OTEL Integrations as they would be dependent on upstream changes

**Review by ishleenk17 — APPROVED** [review]

**Pull request merged**